### PR TITLE
Remove deprecated API calls from display_information

### PR DESCRIPTION
**Pull request by D-R-Smith**
### Description

Update display_information sample to remove use of deprecated API call.

Checklist:
- [ ] Set target branch to main (current release) or v.next (next release)
- [ ] Request a reviewer
- [ ] Assign a reviewer
- [ ] Tag reviewer in comment

### CODE REVIEW

File: display_information/identify-graphics/src/main/java/com/esri/samples/identify_graphics/IdentifyGraphicsSample.java [hunk 1]
```diff
@@ -80,7 +80,7 @@ public void start(Stage stage) {
       mapView.getGraphicsOverlays().add(graphicsOverlay);
 
       // work with the MapView after it has loaded
-      mapView.addSpatialReferenceChangedListener(src -> addGraphicsOverlay());
+      mapView.spatialReferenceProperty().addListener(e -> addGraphicsOverlay());
```
Comment (jenmerritt) [inline display_information/identify-graphics/src/main/java/com/esri/samples/identify_graphics/IdentifyGraphicsSample.java]: I'm not convinced why this is actually being used. The `addGraphicsOverlay` method also says it's adding 4 graphics but it's adding one. I think we could just create the graphics and add them to the graphics overlay before adding the graphics overlay to the mapview. I'll look at this sample in a bit more detail.

**Comment by jenmerritt** [comment]
The deprecated `addSpatialReferenceChangedListener` has been removed but as per my comment above I'm not convinced why it was there. Otherwise I've modularized and updated a Color reference. 
@mbcoder please could you review this one and check you're happy with the changes? Thanks.

**Review by mbcoder — APPROVED** [review]
Yes this looks okay.  I've tried turning off my network connection with testing this to make sure we don't end up with any null pointer exceptions if the basemap doesn't load.  It was fine...

**Comment by jenmerritt** [comment]
Thanks! @Luiy0 second review please

**Review by Luiy0 — APPROVED** [review]
All good.

**Pull request merged**